### PR TITLE
Fix geojson context URI refs

### PR DESCRIPTION
**Pull request by azaroth42**
As reported in #1001.

### CODE REVIEW

File: source/api/annex/services/index.md [hunk 1]
```diff
@@ -140,18 +140,18 @@ With the `logo` property added to the Image Information description in version 2
 
 
 ### 3.2 GeoJSON
-_Added: 2014-05-20_
+_Added: 2014-05-20_, _Latest Revision: 2017-03-29_
 
 [GeoJSON][geojson] provides the ability to associate a geographical place with a resource, in order to drive a map-based user interface or visualization tool.  This might be a location associated with the provenance of the object such as where it was created, or where it is currently held.  The location might also be related to the content of the resource, such as a city mentioned in the text or the landmark depicted in a photograph.  It is not appropriate to use this feature for tagging of time, people, events, and other semantic metadata outside of the scope of the Presentation API.
 
-The [JSON-LD representation][geojson-ld] of GeoJSON, with `@context` `http://geojson.org/contexts/geojson-base.jsonld` _SHOULD_ be used. See the GeoJSON documentation for a full description of the functionality available.
+The [JSON-LD representation][geojson-ld] of GeoJSON, with `@context` `http://geojson.org/geojson-ld/geojson-context.jsonld` _SHOULD_ be used. See the GeoJSON documentation for a full description of the functionality available.
```
Comment (jpstroop) [inline source/api/annex/services/index.md]: Perhaps link GeoJSON to http://geojson.org/ ?

Comment (jpstroop) [inline source/api/annex/services/index.md]: Or, better: http://geojson.org/geojson-ld/

**Comment by jpstroop** [comment]
👍

**Comment by zimeon** [comment]
:+1:

**Comment by tomcrane** [comment]
:+1:

**Comment by mikeapp** [comment]
👍

**Pull request merged**